src/test/java/org/alfresco/repo/security/person/PersonTest.java

```diff
@@ -1788,4 +1789,20 @@ public void testPreventChangesToOtherUsersPropertiesWithoutHavingAdminRights()
             //  expect to go here
         }
     }
+    
+    public void testBuitInSystemUser()
+    {
+
+        AuthenticationUtil.runAsSystem(new RunAsWork<Void>()
+        {
+            @Override
+            public Void doWork()
+            {
+                personService.getPerson(AuthenticationUtil.SYSTEM_USER_NAME);

```

Comment:
Do we need to check if this returns `null`?

Reply:
Now throws an Exception